integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/hybrid/AwsYcloudHybridCloudTest.java

```diff
@@ -161,11 +162,13 @@ public void testCreateSdxOnChildEnvironment(TestContext testContext) {
     }
 
     private void testShhAuthenticationSuccessfull(String username, String host) throws IOException, UserAuthException {
+        String sanitizedUserName = SanitizerUtil.sanitizeWorkloadUsername(username);
         SSHClient client = getSshClient(host);
-        client.authPassword(username, MOCK_UMS_PASSWORD);
+        client.authPassword(sanitizedUserName, MOCK_UMS_PASSWORD);
```

Comment:
Please use sanitizedUserName it the testShhAuthenticationFailure method as well.